resources/default_pipeline_environment.yml

```diff
@@ -1,15 +1,17 @@
 #Project Setup
 general:
   productiveBranch: 'master'
+  kubernetes:
+    jnlpAgent: 'jenkinsci/jnlp-slave:latest'
```

Comment:
can we put this into the step configuration?

using a global section `kubernetes` will likely cause some headaches down the road.
Next to Cloud Foundry we will also have Kubernetes deploy targets. Thus, I would suggest to reserve the name for that.